src/core/server/rendering/types.ts

```diff
@@ -49,6 +50,7 @@ export interface RenderingMetadata {
       translationsUrl: string;
     };
     csp: Pick<ICspConfig, 'warnLegacyBrowsers'>;
+    externalUrl: Pick<IExternalUrlConfig, 'policy'>;
```

Comment:
Nit: it looks like this could be simplified

```suggestion
    externalUrl: IExternalUrlPolicy[];
```